src/app/pages/home/home.component.css

```
@@ -266,8 +267,9 @@ markdown {
 
 /* || Miscellaneous */
 
-#btn-info:hover {
-  background-color: #75797A;
+#btn-info{
+  box-shadow: var(--signing-shadow);
+  background-color: #6C757D;
```

Can this be a variable?
```suggestion
  background-color: var(--accent);
```